lib/setup.coffee

```diff
@@ -81,6 +81,7 @@ module.exports = (app) ->
     app.use require("browserify-dev-middleware")
       src: path.resolve(__dirname, "../")
       transforms: [require("jadeify"), require('caching-coffeeify')]
+      debug: true
```

Comment:
Adds better debugging / sourcemap support: 

Before:

<img width="386" alt="screen shot 2017-06-03 at 7 48 21 pm" src="https://cloud.githubusercontent.com/assets/236943/26758542/c5116674-4895-11e7-9a12-b31b2f092089.png">

After:

<img width="378" alt="screen shot 2017-06-03 at 7 47 49 pm" src="https://cloud.githubusercontent.com/assets/236943/26758543/cb97f184-4895-11e7-9606-9dcc52a48330.png">

Maps to `--debug` in [Browserify](http://browserify.org/):
> Use --debug when creating bundles to have Browserify automatically include Source Maps for easy debugging.